README.md

```diff
@@ -86,13 +86,15 @@ To manage sensitive environment variables:
 
 ### Staging
 
-The app is currently hosted on Heroku: [https://beis-roda-staging.herokuapp.com/](https://beis-roda-staging.herokuapp.com/)
+The app is currently hosted on GPaaS: [https://beis-roda-staging.london.cloudapps.digital](https://beis-roda-staging.london.cloudapps.digital)
 
 The `develop` branch is deployed to staging after a successful build via Travis CI.
 
 ### Production
 
-TBC
+The app is currently hosted on GPaaS: [https://beis-roda-prod.london.cloudapps.digital](https://beis-roda-prod.london.cloudapps.digital)
+
+The `master` branch is deployed to staging after a successful build via Travis CI.
```

Comment:
This still says staging! It should read 
```The `master` branch is deployed to production after a successful build via Travis CI.```

Reply:
I corrected the URLs and missed this bit as well. Updated again!